### PR TITLE
fix(design-tokens): generateTheme now tolerates poorly encoded font stack

### PR DESCRIPTION
**Pull request by lukelooker**
During integration found an issue where `generateTheme` didn't interact well when given more than a single font-face. This change updates `generateTheme` to leverage `sanitizeFontFamily` to make `generateTheme` more tolerant.

This commit also adds `'Noto Sans'` to default fallback font-stack.

### Requirements

Please check the following items are addressed in your pull request (or are not applicable)

- [x] Includes test coverage for all changes
- [ ] Documentation updated
- [x] i18n impacts
- [x] a11y impacts

### CODE REVIEW

File: packages/design-tokens/src/utils/theme.test.ts [hunk 1]
```diff
@@ -99,13 +99,24 @@ describe('generateTheme', () => {
...
   })
 
+  test('fontFamilies - unquoted faces', () => {
+    const generated = generateTheme(theme, {
+      fontFamilies: {
+        body: 'Open Sans, Noto Sans',
+      },
+    })
+    expect(generated.fonts.body).toMatchInlineSnapshot(
+      `"'Open Sans', 'Noto Sans', 'Noto Sans', 'Noto Sans JP', 'Noto Sans CJK KR', 'Noto Sans Arabic UI', 'Noto Sans Devanagari UI', 'Noto Sans Hebrew', 'Noto Sans Thai UI', Helvetica, Arial, sans-serif"`
```
Comment (mdodgelooker) [inline packages/design-tokens/src/utils/theme.test.ts:116]: Is it intentional / ok for `'Noto Sans'` to be in there twice?

File: packages/design-tokens/src/utils/typography/sanitizeFont.test.ts [hunk 2]
```diff
@@ -43,6 +43,18 @@ describe('sanitizeFontFaces', () => {
     expect(sanitizeFontFamily("'Helvetica neue'")).toEqual("'Helvetica neue'")
   })
 
+  it('Several, improperly quoted', () => {
```
Comment (mdodgelooker) [inline packages/design-tokens/src/utils/typography/sanitizeFont.test.ts:46]: Should this be "properly"?
```suggestion
  it('Several, properly quoted', () => {
```

**Review by mdodgelooker — CHANGES_REQUESTED** [review]
Looks good! Just have one question and one test to possibly rename. 🙂

File: packages/design-tokens/src/utils/theme.test.ts (hunk 1 above)
Comment (lukelooker) [reply packages/design-tokens/src/utils/theme.test.ts:116]: In this case yes, (they asked for Noto Sans and it's part of the fallback set by default).

I suppose I could try to de-duplicate the array but it's technically valid and I was aiming to make the fewest possible operations on the input to keep things fast so I thin

File: packages/design-tokens/src/utils/typography/sanitizeFont.test.ts (hunk 2 above)
Comment (lukelooker) [reply packages/design-tokens/src/utils/typography/sanitizeFont.test.ts]: 🤦  - yes, yes it should be. Fixed!

**Review by mdodgelooker — APPROVED** [review]
LGTM!

**Pull request merged**